collector/lib/Logging.h

```diff
@@ -127,11 +127,14 @@ class LogMessage {
 
 #define CLOG(lvl) CLOG_IF(true, lvl)
 
-#define CLOG_THROTTLED_IF(cond, lvl, interval)                                          \
-  static std::chrono::steady_clock::time_point _clog_lastlog_##__LINE__;                \
-  if (collector::logging::CheckLogLevel(collector::logging::LogLevel::lvl) && (cond) && \
-      (std::chrono::steady_clock::now() - _clog_lastlog_##__LINE__ >= interval))        \
-  _clog_lastlog_##__LINE__ = std::chrono::steady_clock::now(),                          \
+#define CLOG_THROTTLED_IF(cond, lvl, interval)                                           \
+  static bool _clog_firstlog_##__LINE__ = true;                                          \
+  static std::chrono::steady_clock::time_point _clog_lastlog_##__LINE__;                 \
+  if (_clog_firstlog_##__LINE__ ||                                                       \
```

Comment:
The `firstlog` boolean seems superfluous, indeed, `std::chrono::steady_clock::time_point` with default constructor represents the clocks epoch.

ref: 
https://en.cppreference.com/w/cpp/chrono/time_point/time_point